instrumentation/redisson/redisson-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/redisson/RedissonRequest.java

```diff
@@ -48,37 +44,30 @@ public String getOperation() {
       CommandsData commandsData = (CommandsData) command;
       if (commandsData.getCommands().size() == 1) {
         return commandsData.getCommands().get(0).getCommand().getName();
+      } else {
+        return "BATCH EXECUTE";
```

Comment:
Does the spec allow this? From what I understand from https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#call-level-attributes operation should not be set if there is more than one operation.